src/Generator.php

```diff
@@ -64,6 +77,11 @@ public function generateClass(string $className, string $templateName, array $va
 
     /**
      * Generate a normal file from a template.
+     *
+     * @param string $targetPath
```

Comment:
This information does not add value because we already have the type in the signature. You should delete them.
BTW, PHP-CS-Fixer should remove it 🤔

Reply:
okay Max